conda_build/render.py

```diff
@@ -78,7 +78,8 @@ def parse_or_try_download(metadata, no_download_source, config,
 
     need_reparse_in_env = False
     if (force_download or (not no_download_source and (metadata.uses_vcs_in_meta or
-                                                       metadata.uses_setup_py_in_meta))):
+                                                       metadata.uses_setup_py_in_meta or
```

Comment:
:+1: to making the changes in fewer places. I only found this by `grep`ping for `load_setup_py_data` to basically duplicate its interface.